src/Castle.Windsor.Tests/LoggingFacility/CustomFacilityTests.cs

```diff
@@ -1,58 +0,0 @@
-﻿// Copyright 2004-2011 Castle Project - http://www.castleproject.org/
-// 
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-// 
-//     http://www.apache.org/licenses/LICENSE-2.0
-// 
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-#if CASTLE_SERVICES_LOGGING
-namespace CastleTests.LoggingFacility
-{
-	using Castle.Core.Logging;
-	using Castle.Core.Resource;
-	using Castle.Facilities.Logging;
-	using Castle.Services.Logging.Log4netIntegration;
-	using Castle.Windsor;
-	using Castle.Windsor.Installer;
-
-	using CastleTests.LoggingFacility.Tests.Classes;
-
-	using NUnit.Framework;
-
-	[TestFixture]
-	public class CustomFacilityTests
-	{
-		[Test]
-		public void ReadCustomFacilityConfigFromXML()
```

Comment:
What is wrong with allowing people to configure the logging facility by XML? `loggingApi` is the only property listed in the changelog that is being deprecated as it is assumed to be "custom".

Reply:
We also need to start a discussion issue about possibly removing XML configuration in v5, we should keep things consistent, either all the facilities should support it or none should.

Reply:
Will refer back to issue on this and flesh this out with more discussion from our call.